dallinger/experiment_server/experiment_server.py

```diff
@@ -319,7 +319,10 @@ def advertisement():
         status = part.status
     except exc.SQLAlchemyError:
         status = None
-
+    try:
+        app_id = config.get('id')
```

Comment:
I'm on board with the idea of having default values baked into the configuration manager itself; that's an improvement. The goal here is slightly different, though — it's not to have a default value for the parameter, but a default behavior when the parameter hasn't been defined.